meeting_minutes/220113_SARIF_TC_50.md

```diff
@@ -0,0 +1,162 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company          | Role(s)                 |
+| :--------- | :-------- | :--------------- | :---------------------- |
+| Aditya     | Sharad    | Microsoft        | Member                  |
+| Courtney   | Lawton    | Microsoft        | Public Participant      |
+| David      | Keaton    | Individual       | Chair                   |
+| Eddy       | Nakamura  | Microsoft        | Voting member           |
+| James      | Spoor     | Snyk Ltd         | Member                  |
+| Katrina    | O'Neil    | Micro Focus      | Voting member           |
+| Nathan     | Baird     | Microsoft        | Member                  |
+| Paul       | Anderson  | Grammatech, Inc. | Voting member           |
+| Stefan     | Hagen     | Individual       | Secretary, taking notes |
```

Comment:
just noticed: wasn't Michael present at the meeting as well?

Reply:
Yes, he was.  Sorry I missed this.  This is a correction we should mention during approval of the minutes.